templates/protostar/error.php

```diff
@@ -68,6 +68,9 @@
 		</style>
 	<?php endif; ?>
 	<link href="<?php echo $this->baseurl; ?>/templates/<?php echo $this->template; ?>/css/template.css" rel="stylesheet" />
+	<?php if (@filesize('templates/' . $this->template . '/css/user.css') > 5): ?>
```

Comment:
```suggestion
	<?php if (file_exists('templates/' . $this->template . '/css/user.css')) : ?>
```
Why not use file_exists?

Reply:
can do if you want. I do it this way for my clients as they often add a user.css file and then remove all the content but dont delete the file.

Reply:
I see...so only load it if there is content?

Reply:
Exactly

Reply:
Please use file_exists, suppressing the error is really bad and shouldn't be used in core if possible.

Reply:
It is not suppressing an error at all

Reply:
Thats what @ does

Reply:
The other pages except for the error page link to this file even when empty. Thus it is not consistent on every page.